Gemfile.lock

```diff
@@ -1,60 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.5.0)
+    better_errors (2.5.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (10.0.2)
+    byebug (11.0.1)
     coderay (1.1.2)
     commander (4.4.7)
       highline (~> 2.0.0)
     database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubi (1.7.1)
-    event_sourcery (0.22.0)
+    erubi (1.9.0)
+    event_sourcery (0.23.0)
     event_sourcery-postgres (0.8.0)
       event_sourcery (>= 0.14.0)
       pg
       sequel (>= 4.38)
-    highline (2.0.0)
+    highline (2.0.3)
     method_source (0.9.2)
     mustermann (1.0.3)
     pg (0.20.0)
     pry (0.12.2)
       coderay (~> 1.1.0)
       method_source (~> 0.9.0)
-    pry-byebug (3.6.0)
-      byebug (~> 10.0)
+    pry-byebug (3.7.0)
+      byebug (~> 11.0)
       pry (~> 0.10)
-    rack (2.0.6)
-    rack-protection (2.0.4)
+    rack (2.0.8)
+    rack-protection (2.0.7)
```

Comment:
### [rack-protection](http://sinatrarb.com/protection/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.0.4 → 2.0.7

[_~~change-log~~, [source-code](https://github.com/sinatra/sinatra/tree/master/rack-protection)_]





<details>
<summary>Commits</summary>

A change of **92** commits. See the full changes on [the compare page](https://github.com/sinatra/sinatra/compare/v2.0.4...v2.0.7).

These are the first 10 commits:
- (cb45e46) [Add flush option to content_for](https://github.com/sinatra/sinatra/commit/cb45e46fdb51c683f431a6fbc4558379f5b1689c)
- (7989a78) [Drop activesupport dependency](https://github.com/sinatra/sinatra/commit/7989a78392a7ea814a9644d1216f0355deda28c5)
- (b7c1064) [prefer be_include over have_key in this case](https://github.com/sinatra/sinatra/commit/b7c10649cad56f3263df089027621fb1753620cb)
- (e874a76) [Translating readme to malayalam. Half of the portion is done](https://github.com/sinatra/sinatra/commit/e874a766aa1f22032ce53ce13f0a07f88fa6eb00)
- (e3d96ee) [Revert back to using include matcher](https://github.com/sinatra/sinatra/commit/e3d96ee91fb3ab797d54595f972613686ae0cb84)
- (91ba82e) [IndifferentHash monkeypatch warning improvements](https://github.com/sinatra/sinatra/commit/91ba82ec512351db9438e5001fcbbe48f30f5bfd)
- (d5bef3d) [Merge pull request #1482 from jkowens/master](https://github.com/sinatra/sinatra/commit/d5bef3d210de4fdf5844e907eb20d2169dfb80b6)
- (b6790a0) [Merge pull request #1481 from anusreePrakash/master](https://github.com/sinatra/sinatra/commit/b6790a01b83f930df7fe7983583020575dd1127e)
- (e0aa37d) [Merge pull request #1448 from yob/remove-activesupport](https://github.com/sinatra/sinatra/commit/e0aa37dce6f25700131ced4a09e10dda67250b36)
- (8045fbe) [upgrade ruby versions](https://github.com/sinatra/sinatra/commit/8045fbe69f02a8184251adb9e23b6aec41b19281)


</details>